### PR TITLE
feat : build home page

### PR DESCRIPTION
**Pull request by Shubhamkr2610**
Created Home page

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    background-color: var();
+}
+a{
+    text-decoration: inherit;
+    color: inherit;
+}
+.app{
+    min-height: 100vh;
+    /* background-color: var(--clr-page-light-bg); */
+}
```
Comment (Rockyraven) [inline src/App.css:20]: remove that unwanted code

File: src/components/header/Header.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    </Link>
+    <div className='searchbar-container' >
+    <input placeholder='search videos '/>
+
+    </div>
+ 
+
+    <Link to="/login" className='login-button'>
+    <PersonIcon/>
+    </Link>
```
Comment (Rockyraven) [inline src/components/header/Header.js:25]: remove unwanted space

File: src/components/header/header.css [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+        height: 20px;
+        width: 20px;
+    }
+    .searchbar-container{
+        /* display: none; */
+        /* width: 10%; */
+        /* justify-content: center; */
+        /* align-items: center; */
+    }
+}
```
Comment (Rockyraven) [inline src/components/header/header.css:85]: remove this too

**Review by Rockyraven — APPROVED** [review]
remove all unwanted code and do some minar changes
otherwise your code is clean

**Review by arjun-computer-geek — APPROVED** [review]
Good Folder structure. Understandable code.

**Pull request merged**